packages/wallet/src/redux/protocols/virtual-funding/reducer.ts

```diff
@@ -34,7 +40,7 @@ export function initialize(sharedData: SharedData, args: states.InitializationAr
     commitmentType: CommitmentType.PreFundSetup,
     clearedToSend: true,
     processId,
-    protocolLocator: states.GUARANTOR_CHANNEL_DESCRIPTOR,
+    protocolLocator: ADVANCE_CHANNEL_PROTOCOL_LOCATOR,
```

Comment:
Should this be `makeLocator(EmbeddedProtocol.VirtualFunding,EmbeddedProtocol.AdvanceChannel)`?